src/plugins/gui/components/GuiFormField.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react'
+import { View } from 'react-native'
+
+import { useTheme } from '../../../components/services/ThemeContext'
+import { OutlinedTextInput, OutlinedTextInputRef, OutlinedTextInputReturnKeyType } from '../../../components/themed/OutlinedTextInput'
+import { FORM_FIELD_DISPLAY_PROPS, FormFieldType } from '../../../types/FormTypes'
+
+// TODO: Consolidate with FormField
+
+interface Props {
+  label: string
+  fieldType?: FormFieldType
+  key?: string
+  autofocus?: boolean
+  returnKeyType?: OutlinedTextInputReturnKeyType
+  fieldRef?: React.Ref<OutlinedTextInputRef> | undefined
+  value?: string
+  onChangeText: ((text: string) => void) | undefined
+  onFocus?: (() => void) | undefined
+  onBlur?: (() => void) | undefined
+}
+
+export const GuiFormField = React.memo((props: Props) => {
```

Comment:
For another PR: Remove file as abstraction not adding a lot of value

Reply:
https://app.asana.com/0/1200382638405084/1204355837318784/f